local/Makefile.am

```diff
@@ -32,6 +32,10 @@ if TARGET_APPLE
 libopenwebrtc_local_la_SOURCES += owr_device_list_avf.m
 endif
 
+if OWR_GST
+libopenwebrtc_local_la_SOURCES += owr_gst_local_media_source.c
```

Comment:
I thought gst/ only gets sources for OwrGst library.
OwrGstLocal is a part of the main library, so I put it in local/.